app/views/my_adventures/index.html.erb

```diff
@@ -60,6 +65,59 @@
         <% end %>
       </div>
 
+      <div class="card mt-4 mb-4 p-4">
+        <h2>MY COMPLETED ADVENTURES (<%= @done_adventures.count %>)</h2>
+        <%= "Go on some adventures!" if @done_adventures.empty?  %>
+
+        <% @done_adventures.each_with_index do |a, index| %>
+          <div class="d-inline-flex align-items-center">
+            <%= link_to adventure_path(a.adventure), class: "card-product", id: "card-link-text" do %>
+              <% if a.adventure.photo.key.present? %>
```

Comment:
I get an error here on line 75 when I try to load the my_adventures page. 
`Module::DelegationError in MyAdventures#index`

Error message says this:
`key delegated to attachment, but attachment is nil`

Reply:
The page works now and the button works as expected. :) Nice!